router/mux/engine.go

```diff
@@ -11,7 +11,7 @@ import (
 // Engine defines the minimun required interface for the mux compatible engine
 type Engine interface {
 	http.Handler
-	Handle(pattern string, handler http.Handler)
+	Handle(pattern, method string, handler http.Handler)
```

Comment:
this change to the interface won't let you use the `http.NewServeMux()` implementation (that's why the tests are crashing), so we need to find a workaround